src/js/helpers/elementCreatorHelper.js

```diff
@@ -78,7 +85,95 @@ export function createButton(args) {
 ];
   const btn = document.createElement('button');
   btn.innerText = btnText;
+  btn.id = btnId;
   btn.classList.add(...allClasses);
 
   return btn;
 }
+
+/**
+ * Create a headband to select the number of players
+ * @param {String} arg The minimum number of players
+ * @returns {DOM element} headband The headband 'span' element, containing +/- buttons and displaying the number of players
+ */
+export function createNbOfPlayers(arg){
+  
+  const minusBtn = createButton({
+    btnText: '-',
+    btnId: 'remove',
+  });
+
+  const plusBtn = createButton({
+    btnText: '+',
+    btnId: 'add',
+  });
+  
+  const playersCount = document.createElement('div');
+  const countClasses = [
+    'text-xl',
+    'text-center',
+    'h-full',
+    'leading-[3]',
+  ];
+  playersCount.textContent = arg;
+  playersCount.id = 'nbOfPlayers';
+  playersCount.classList.add(...countClasses);
+
+  const headbandClasses = [
+    'inline-flex',
+    'justify-evenly',
+  ];
+  
+
+  const headband = document.createElement('span');
+
+  headband.classList.add(...headbandClasses);
+  headband.append(
+    minusBtn,
+    playersCount,
+    plusBtn
+  );
+
+  return headband;
+}
+
+/**
+ * Create a list of inputs for players' names
+ * @param {Number} args The number of players
+ * @returns {DOM element} subContainer The list of 'div' editable elements for players' names
+ */
+export function createNames(args){
+  const subContainer = createContainer({
+    classes: [
+      'p-8',
+      'm-8',
+      'border-solid',
+      'border-2',
+      'rounded',
+      'flex',
+      'flex-col',
+      'justify-center'
+    ]
+  });
+
+  for(let i = 1; i <= args; i++){
+
+    const player = document.createElement('div');
+    player.contentEditable = true;
```

Comment:
Yeah, I didn't use `input` elems mainly because they're ugly and difficult to style with CSS (unless there exist some tricks I don't know about?)

Since I'm new to back-end stuff and never used a form before, I lack the long-term insight that you must have: is it impossible (or overly complex) to submit the info using `div` elems instead of input/form ? If that's the case, I'll go ahead and make the changes !

Reply:
It's completely possible to submit the data to the backend using a div - but you'd essentially have to re-implement a feature you get for free with a html form using javascript (so kind of a waste of effort). The other thing to bear in mind is accessibility - using html elements in the way they're intended is important for the proper functioning of things like screen-readers. 

For the styles, it's just a case of overriding the browser defaults, or we can even add a css file which resets all the browser defaults so they stop annoying us (do a bit of googling around reset.css and you'll probably find an example we can borrow from).

Reply:
Gotcha, I'll switch to form/input elems

Reply:
Done ! Waiting for merge.
No idea where the indentation problem comes from though, on my VS-code the indentation is fine, no extra tabs or anything... weird.

Reply:
Nice one!

We probably don't have the same settings in our vscode - I'll double check what mine are and we'll make sure they're lined up :)